java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java

```diff
@@ -301,4 +303,82 @@ void testQuantilesOnDoubleInput() {
       }
     }
   }
+
+  @Test
+  void testSliceWithArray() {
+    assumeTrue(Cuda.isEnvCompatibleForTesting());
+    try(ColumnVector cv = ColumnVector.fromBoxedInts(10, 12, null, null, 18, 20, 22, 24, 26, 28)) {
+      //{{12, null}, {20, 22, 24, 26}, {14, 16}, {}}
```

Comment:
Nit: This line can be removed. Same comment for other test as well.

Reply:
PTAL